app/views/posts/index.html.erb

```diff
@@ -1,2 +1,20 @@
-<h1>All Posts for a Given User</h1>
-<p>Here is a list of posts for a given user.</p>
+<div class="container mt-3">
+  <%= image_tag(@user.photo, alt: @user.name, class: 'user-image') %>
+  <h2 class="d-inline-block"><%= @user.name %></h2>
+  <p class="right">Number of Posts: <%= @user.posts.count %></p>
+</div>
+<% @posts.each do |post| %>
+  <div class="jumbotron shadow p-5">
+    <h4><%= link_to post.title, user_post_path(@user, post), class: 'text-decoration-none text-dark' %></h4>
+    <p class="right">Comments: <%= post.comments.count %> Likes: <%= post.likes.count %></p>
+    <% if post.comments.any? %>
+      <div class="jumbotron shadow p-5">
+        <% post.comments.each do |comment| %>
+          <p><%= comment.user.name %>: <%= comment.text %></p>
+        <% end %>
+      </div>
+    <% end %>
+  </div>
+<% end %>
```

Comment:
- [ ] Great effort so far 👏✨ However, kindly make sure to use the `recent_comments` method present in `post.rb` to render 5 most recent comments on post index page as [displayed](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_user_all_posts.png?token=GHSAT0AAAAAAB5V373UCMRN55B3KJOJG2WAZLQVKZQ)

Current UI | Wireframe
---| ---|
![image](https://github.com/jkanyi-web/Blog-App/assets/106119105/ec370664-ce57-4179-bc08-72fdcdbe6409) | ![image](https://github.com/jkanyi-web/Blog-App/assets/106119105/f2f413b4-4aa5-4576-b001-7ee1a44054db)